website_snippet_anchor/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Set Snippet's Anchor",
+    "summary": "Allow to reach a concrete section in the page",
+    "version": "8.0.1.0.0",
+    "category": "Website",
+    "website": "http://www.antiun.com",
+    "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```

Comment:
Hi @Yajo 
Thanks for your contribution
I will check mostly format and guidelines in your PR
Can you remove the empty keys?